src/components/players-online-title/players-online-title.jsx

```diff
@@ -0,0 +1,11 @@
+import './players-online-title.scss';
+
+function PlayersOnlineTitle({ playerNum = 1 }) {
```

Comment:
there is setInterval hook in Homepage screen which fetch players count from backend, move this logic to PlayersOnlineTitle component